src/language-service/src/schemas/sensors.ts

```diff
@@ -7,50 +6,51 @@ export type SensorFile = Sensors | Sensors[];
  * @TJS-additionalProperties true
  */
 export interface OtherSensor {
-    /**
-     * @TJS-pattern (^(?!.*(mqtt|template)).*)$
-     */
-    platform: string;
+  /**
+   * @TJS-pattern (^(?!.*(mqtt|template)).*)$
+   */
+  platform: string;
 }
 
 export interface MqttSensor {
-    platform: "mqtt";
-    state_topic: string;
-    name?: string;
-    qos?: number;
-    unit_of_measurement?: string;
-    icon?: string;
-    expire_after?: number;
-    value_template?: string;
-    force_update?: boolean;
-    availability_topic?: string;
-    payload_available?: string;
-    payload_not_available?: string; 
-    json_attributes?: string | string[];
-    json_attributes_template?: string;
-    json_attributes_topic?: string;
-    unique_id?: string;
-    device_class?: string;
-    device?: any;
+  platform: "mqtt";
+  state_topic: string;
+  name?: string;
+  qos?: number;
+  unit_of_measurement?: string;
+  icon?: string;
+  expire_after?: number;
+  value_template?: string;
+  force_update?: boolean;
+  availability_topic?: string;
+  payload_available?: string;
+  payload_not_available?: string;
+  json_attributes?: string | string[];
+  json_attributes_template?: string;
+  json_attributes_topic?: string;
+  unique_id?: string;
+  device_class?: string;
+  device?: any;
+  scan_interval: number;
```

Comment:
nullable

Reply:
scan_interval is not nullable. You either set it, or you don't. No value != value for the schema of Home Assistant.

Reply:
Oh right... now I see.

Reply:
Fixed in https://github.com/keesschollaart81/vscode-home-assistant/pull/374/commits/37e6dae25f0881a73d18f1ddac58d775c91ec535